athena-cloudera-impala/src/main/java/com/amazonaws/athena/connectors/cloudera/ImpalaEnvironmentProperties.java

```diff
@@ -0,0 +1,39 @@
+/*-
+ * #%L
+ * Amazon Athena Query Federation SDK
+ * %%
+ * Copyright (C) 2019 - 2024 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.athena.connectors.cloudera;
+
+import com.amazonaws.athena.connectors.jdbc.JdbcEnvironmentProperties;
+
+import java.util.Map;
+
+public class ImpalaEnvironmentProperties extends JdbcEnvironmentProperties
+{
+    @Override
+    protected String getConnectionStringPrefix(Map<String, String> connectionProperties)
+    {
+        return "impala://jdbc:impala://";
+    }
+
+    @Override
+    protected String getDatabase(Map<String, String> connectionProperties)
+    {
+        return "/";
```

Comment:
if database is empty why are we returning '/'?

Reply:
https://github.com/awslabs/aws-athena-query-federation/blob/b54a0216a2f1efbcd9ac226411684ac7828160cf/athena-jdbc/src/main/java/com/amazonaws/athena/connectors/jdbc/JdbcEnvironmentProperties.java#L51C1-L55C1

getDatabase is setup to add separator + DATABASE parameter.

Since Impala (https://docs.aws.amazon.com/athena/latest/ug/connectors-cloudera-impala.html) does not require DATABASE or have an example with it, i remove that parameter from being added